sdk/azcore/policy/policy.go

```diff
@@ -126,3 +126,10 @@ func WithHTTPHeader(parent context.Context, header http.Header) context.Context
 func WithRetryOptions(parent context.Context, options RetryOptions) context.Context {
 	return context.WithValue(parent, shared.CtxWithRetryOptionsKey{}, options)
 }
+
+// IncludeResponse applies the HTTP response retrieval annotation to the parent context.
+// Call runtime.ResponseFromContext() to retrieve the HTTP response from the context.
+func IncludeResponse(parent context.Context) context.Context {
```

Comment:
Note that I changed the name from `WithRawResponse` to `IncludeResponse`.  I felt that using the `With` prefix was misleading in cases where we're enabling a behavior (I also removed `Raw` to shorten the name).  Maybe this should be `EnableResponse`?

Reply:
I actually prefer `Include`, `Add` or even `With` because this func causes the policy to augment the context with something it wouldn't otherwise have.

Reply:
Will keep it as `Include`.